### ISSUE TITLE
"Unhandled promise rejection: No cast session"

### ISSUE THREAD

**Issue by mhamann** [flextv/react-native-google-cast#13]
If I start casting something, leave the app, come back, and try to play something else, the Cast Button seems to know that the device is still connected, but it seems that it doesn't know about the active cast session, so calling `castMedia()` fails.

If the status is still "Connected", then the library should re-establish the Cast session.

**Comment by jedashford** [comment]
Yeah, we just found this bug too (verified issue for iOS, haven't tested on Android yet.
